projects/batfish-client/src/main/java/org/batfish/client/Command.java

```diff
@@ -249,8 +249,8 @@
     descs.put(
         LOAD_QUESTIONS,
         new Pair<>(
-            "<path to directory containing question json files",
-            "Load questions from library directory"));
+            "[-loadremote] <path to directory containing question json files>",
```

Comment:
Confirm that this matches the expected semantics. Is the second argument is still required?

Explain better what happens if both arguments are supplied.